test/JSON/toJSON.js

```diff
@@ -15,12 +15,141 @@ fnc.prototype.toJSON = function() {
   return this.number.toString();
 }
 
+// test getter - no-enum
+{
+  var obj = {};
+  Object.defineProperty(obj, "c", { get: () => 3, enumerable: false });
+
+  obj.a = 1;
+  obj.b = 2;
+  obj.d = 4;
+  TEST('{"a":1,"b":2,"d":4}', JSON.stringify(obj));
```

Comment:
interesting to see that we didn't have any test which does stringify on getter/setter object. Thanks for adding it.

Reply:
That reminds me actually; can you please add a test case where a single property has both a getter and a setter? That specifically is what caused issues for node, since it meant one property used up more than one slot.